assume/common/outputs.py

```diff
@@ -359,6 +359,8 @@ async def on_stop(self):
                 df = pd.read_sql(query, self.db)
             except (OperationalError, ProgrammingError):
                 continue
+            except Exception as e:
+                raise e
```

Comment:
This does not do anything - it just catches and raises again.
I think you need this:

```suggestion
            except Exception as e:
                logger.error("could not read query: %s", e)
                continue
```